tools/congestion-model/src/lib.rs

```diff
@@ -0,0 +1,23 @@
+pub mod designs;
+mod evaluation;
+mod model;
+pub mod workload;
+
+pub use designs::Shard;
+pub use evaluation::{summary_table, TransactionStatus};
+pub use model::{Model, Queue, QueueId, Receipt, ShardId, TransactionId};
+pub use workload::{ReceiptDefinition, ReceiptId, TransactionBuilder};
+
+pub(crate) use model::Transaction;
+
+/// Gas is measured in Giga Gas as the smallest unit. This way, it fits in a u64
+/// even for long simulations.
+type GGas = u64;
+type Round = u64;
+
+pub const GGAS: GGas = 10u64.pow(0);
+pub const TGAS: GGas = 10u64.pow(3);
+pub const PGAS: GGas = 10u64.pow(6);
+
+pub const TX_GAS_LIMIT: GGas = 500 * TGAS;
+pub const GAS_LIMIT: GGas = 1000 * TGAS;
```

Comment:
It would help if these references which protocol configurations and/or constants these refer to in case somebody from the future needed to modify these.

Reply:
Good point, I've added comments describing where these values come from.